src/main/java/create_task_use_case/CreateTaskController.java

```diff
@@ -15,7 +15,22 @@ public CreateTaskController(CreateTaskInputBoundary inputBoundary) {
         this.inputBoundary = inputBoundary;
     }
 
-    CreateTaskOutputData createTask(String title, int dayIndex){
+    CreateTaskOutputData createTask(String title, String day){
+        int dayIndex = 0; // if no match assume sunday
+        switch (day){
+            case "Monday":
+                dayIndex = 1;
+            case "Tuesday":
+                dayIndex = 2;
+            case "Wednesday":
+                dayIndex = 3;
+            case "Thursday":
+                dayIndex = 4;
+            case "Friday":
+                dayIndex = 5;
+            case "Saturday":
+                dayIndex = 6;
+        }
```

Comment:
This should happen in inputData

Reply:
I have made the desired changes, Please check my new push.